in_toto/in_toto_record.py

```diff
@@ -172,6 +176,12 @@ def main():
       " link metadata."
       " (passing one of '--key' or '--gpg' is required)"))
 
+  parent_parser.add_argument("-t", "--key-type", dest="key_type",
+      type=in_toto.util.KeyType, choices=list(in_toto.util.KeyType),
```

Comment:
I was expecting the type to be `str` (e.g., 'rsa').  Isn't `in_toto.util.KeyType` a `class KeyType(enum.Enum)`?